src/WooCommerce.php

```diff
@@ -750,4 +773,58 @@ public static function getProductAttributes(\WC_Product $product) {
     return $data;
   }
 
+  /**
+   * Checks if product GTIN is found for any other product IDs.
+   *
+   * @param int $product_id
+   *   Product ID.
+   *
+   * @param int $gtin
+   *   GTIN.
+   *
+   * @return bool
+   */
+  public static function is_existing_gtin($product_id, $gtin) {
+    global $wpdb;
+
+    return $wpdb->get_var(
+      $wpdb->prepare(
+        "
+        SELECT pm.post_id
+        FROM {$wpdb->postmeta} pm
+        WHERE pm.meta_key = %s
+        AND pm.meta_value = %s
+        AND pm.post_id <> %d
+        LIMIT 1
+        ",
+        '_' . Plugin::PREFIX . '_gtin',
+        $gtin,
+        $product_id
+      )
+    );
+  }
+
+  /**
+   * Wraps is_existing_gtin for responding to ajax requests.
+   */
```

Comment:
Please add an extra comment line with `@implements wp_ajax_is_existing_gtin`. Leave a blank line before it. :)